### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jgeudens**
This MR will update the documentation (user manual) to the latest state of the application, including new screenshots.

The text will be optimised using ChatGPT.

### CODE REVIEW

**Comment by jgeudens** [comment]
@ben-edna
Do you want to review the documentation? 

* Online version: https://modbusscope.readthedocs.io/en/doc/#
* Pdf version (CI artifact): https://github.com/jgeudens/ModbusScope/actions/runs/4068049088
* Updated animated gif: https://github.com/jgeudens/ModbusScope/blob/doc/modbusscope_demo.gif

File: docs/pages/importing_and_exporting.md [hunk 1]
```diff
@@ -2,20 +2,14 @@
 
 ## Storing and reusing configuration
 
-The configuration of registers as described in the previous section can be stored and reloaded in a project file. Storing the settings in a *.mbs* file can be done through the *File > Save Project As...* and reloading them can be done through *File > Open Project ...* or by dragging a *.mbs* file into the application.
+The configuration of registers, as described in previous sections, can be saved and loaded in a project file. These settings can be saved in a *.mbs* file by going to *File > Save Project As...* and loaded by going to *File > Open Project...* or by dragging a *.mbs* file into the application.
```
Comment (spoorcc) [inline docs/pages/importing_and_exporting.md:5]:
```suggestion
The configuration of registers, as described in previous sections, can be saved and loaded in a project file. These settings can be saved in a `.mbs` file by going to *File > Save Project As...* and loaded by going to *File > Open Project...* or by dragging a `.mbs` file into the application.
```

File: docs/pages/importing_and_exporting.md [hunk 2]
```diff
@@ -2,20 +2,14 @@
...
 
 ## Exporting data/image
 
-Current log results can be exported either as image or as data (*.csv*) file. This can be selected with either *File > Save Data File As...* or  *File > Export Image As...* . Saving a project/data file or exporting an image is only possible when logging isn't active.
-
-## Open data file
-
-Opening a data file is described in detail in a separate chapter of the user manual.
+Current log results can be exported as an image or as data (*.csv*) file. You can select either *File > Save Data File As...* or *File > Export Image As...* to do so. It is important to note that saving a project/data file or exporting an image can only be done when logging is not active.
```
Comment (spoorcc) [inline docs/pages/importing_and_exporting.md:9]:
```suggestion
Current log results can be exported as an image or as data (`.csv`) file. You can select either *File > Save Data File As...* or *File > Export Image As...* to do so. It is important to note that saving a project/data file or exporting an image can only be done when logging is not active.
```

File: docs/pages/importing_and_exporting.md [hunk 3]
```diff
@@ -2,20 +2,14 @@
...
 
 ## Import register definitions from *mbc* file
 
-*ModbusControl* is an application to read/write registers from Modbus slaves. The *ModbusControl* project file (*.mbc*) can be used to add registers definitions to *ModbusScope*. The easiest way is to click on *Import from .mbc file* in the register dialog. You can also open this window by dragging and dropping the *.mbc* file in the main screen of *ModbusScope*.
+*ModbusControl* is an application that can be used to read and write data from Modbus slaves. It is possible to import the register definitions from a *ModbusControl* project file (*.mbc*) into *ModbusScope* by clicking on *Import from .mbc file* in the register dialog or by dragging and dropping the *.mbc* file into the main screen of *ModbusScope*. This makes it easy to add register definitions to *ModbusScope* from a *ModbusControl* project file.
```
Comment (spoorcc) [inline docs/pages/importing_and_exporting.md:13]:
```suggestion
*ModbusControl* is an application that can be used to read and write data from Modbus slaves. It is possible to import the register definitions from a *ModbusControl* project file (`.mbc`) into *ModbusScope* by clicking on *Import from .mbc file* in the register dialog or by dragging and dropping the `.mbc` file into the main screen of *ModbusScope*. This makes it easy to add register definitions to *ModbusScope* from a *ModbusControl* project file.
```

File: docs/pages/open_data_file.md [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
+When a data file that was created by *ModbusScope* is opened, the data will be loaded automatically. This can be done by selecting *File* > *Open Data File...* or by dragging the file into the application. If the file format is not compatible with *ModbusScope*, the program will try to automatically determine the necessary settings for parsing the file. This includes the field and decimal separators. These settings can be adjusted manually if needed. Once the settings are configured, the rest of the file will be loaded and the data can be viewed in the graph as with a normal *ModbusScope* log.
 
 ![image](../_static/user_manual/import_csv.png)
 
 ## Parse settings
 
-There isn't a standard for the format of a *.csv* file. Parsing a data file is only possible when all settings are correct for that specific data file. There are several categories of settings. The settings can be changed and the grid will be updated with the parsing results.
+The format of a *.csv* file can vary, so correct settings must be in place for parsing the file. These settings can be adjusted and the results of the parsing will be reflected in the grid display.
```
Comment (spoorcc) [inline docs/pages/open_data_file.md:9]:
```suggestion
The format of a `.csv` file can vary, so correct settings must be in place for parsing the file. These settings can be adjusted and the results of the parsing will be reflected in the grid display.
```

File: docs/pages/open_data_file.md [hunk 5]
```diff
@@ -27,7 +23,7 @@ Depending on the configured region, the format of the data will will be differen
 
 ### File structure related
 
-*ModbusScope* can only parse *.csv* files which are in a similar format of the *ModbusScope* export files. This means a timestamps column followed by one or more data columns.
+*ModbusScope* is able to process and read data from a *.csv* file if it's formatted in a similar way to the files exported by *ModbusScope*. This includes having a column for timestamps and one or more columns for data.
```
Comment (spoorcc) [inline docs/pages/open_data_file.md:26]:
```suggestion
*ModbusScope* is able to process and read data from a `.csv` file if it's formatted in a similar way to the files exported by *ModbusScope*. This includes having a column for timestamps and one or more columns for data.
```

File: docs/pages/overview.md [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
+This guide explains the basics of using ModbusScope. The software is designed to collect and display data from Modbus registers in a graphical format, making it easy to monitor and analyze the dynamic behavior of a system.
 
 ## Installing
 
-The *ModbusScope* installer or standalone version can always be downloaded from the [release page](https://github.com/jgeudens/ModbusScope/releases).  
+The *ModbusScope* installer or standalone version can always be downloaded from the [release page](https://github.com/jgeudens/ModbusScope/releases).
 
-*ModbusScope* can be installed with the provided *.msi* installer. Double-click it and follow the instructions. The installer will install the required files on your computer. At the end of the install process, you can also select to open *.mbs* files with *ModbusScope* by default.
+ModbusScope can be easily installed by double-clicking on the provided .msi installer file and following the on-screen instructions. The installer will handle the installation of all necessary files on your computer. At the end of the install process, you have the option to set ModbusScope as the default application for opening .mbs files.
```
Comment (spoorcc) [inline docs/pages/overview.md:9]:
```suggestion
ModbusScope can be easily installed by double-clicking on the provided `.msi` installer file and following the on-screen instructions. The installer will handle the installation of all necessary files on your computer. At the end of the install process, you have the option to set ModbusScope as the default application for opening `.mbs` files.
```

File: docs/pages/graphview.md [hunk 7]
```diff
@@ -1,41 +1,48 @@
...
 
 ### Y1- and Y2-axis
 
-Compared to the x-axis setting, the y-axis shares 2 settings: *full auto-scale* and *manual*. These settings are the same as with the x-axis. The 2 other modes are *window auto-scale* and the *limit from* setting. The *window auto-scale* automatically scales the y-axis based on the values that are currently visible. The *limit from* setting allows the user to configure the minimum and maximum values of the axis. There are 2 Y-axis available. An Y-axis can be selected per graph.
+Compared to the x-axis, the y-axis has two similar settings: *full auto-scale* and *manual*. These options work the same way as they do for the x-axis. In addition to these, the y-axis also has two other modes: *window auto-scale* and the *limit from* setting.
+
+* *Window auto-scale* automatically adjusts the range of the y-axis based on the values currently visible in the graph.
+* *Limit from* setting allows the user to set the minimum and maximum values for the y-axis.
+
+There are 2 Y-axis available, one for each graph, and the user can select which Y-axis to use per graph.
```
Comment (spoorcc) [inline docs/pages/graphview.md:30]: Maybe add how?

**Review by spoorcc — CHANGES_REQUESTED** [review]
Nice job! Added some minor formatting comments

**Comment by jgeudens** [comment]
> Nice job! Added some minor formatting comments

Thanks

**Pull request merged**